client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ReachContact.tsx

```diff
@@ -29,11 +29,11 @@ const ReachContact = (props: Props) => {
 
     const watchContactStatus = watch(`form[${index}].contactStatus`)
     const currentContactStatus = watchContactStatus ? watchContactStatus : formValues.contactStatus;
-
-    const currentValue = contactStatuses.find(
+    const foundValue = contactStatuses.find(
         (contactStatus: ContactStatus) =>
             contactStatus.id === currentContactStatus
     );
+    const currentValue = foundValue || { id : -1 , displayName : "..."}
```

Comment:
why displayName is `...` and not just `''`?

Reply:
🤷‍♂️ , its not really shown anyway

Reply:
ok

Reply:
+it'll be easier to debug in test